firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java

```diff
@@ -337,6 +337,22 @@ public void fetchWithExpiration_cacheHasNotExpired_doesNotFetchFromBackend() thr
     verifyBackendIsNeverCalled();
   }
 
+  @Test
```

Comment:
Nice realtime tests! 😄  Mind splitting the ones that aren't for the fetch id into their own PR to make this a bit easier to review? Also, I'm not sure I see a test for the header being set?

Reply:
Removed and will move to a separate PR